src/duct.py

```diff
@@ -153,7 +155,7 @@ def create_and_parse_args():
         help="Interval in seconds between status checks of the running process.",
     )
     parser.add_argument(
-        "--output_prefix",
+        "--output-prefix",
         type=str,
         default=os.getenv("DUCT_OUTPUT_PREFIX", f".duct/run-logs/{file_safe_iso}"),
         help="Directory in which all logs will be saved.",
```

Comment:
- don't make it `f` right here -- keep it as a template
- extend doc to describe which variables are supported, e.g.  `file_safe_iso`, `session_id`
- make default to be `{file_safe_iso}-{session_id}_`

Reply:
> don't make it f right here -- keep it as a template

i don't follow rephrase?

Reply:
keep it as 

```
default=os.getenv("DUCT_OUTPUT_PREFIX", ".duct/run-logs/{file_safe_iso}")
```

so the value would be a template for `.format()` call so user could say e.g. `--output-prefix mylogs/{file_safe_iso}-{session_id}/`

Reply:
got it thanks

Reply:
I disagreed already -- I think we need to give flexibility to the user.
IMHO there should be little of complexity whatsoever.  As soon as you know session (without session you do not start anything) -- you are ready to format the path, so just do it and use that formatted `output_prefix_formated` from that point on. What complexity do you have in mind?

Reply:
TeeStream needs to calculate the file path it will be writing to, and open the file prior to executing Popen.

Reply:
so? decide then on the path before creating TeeStreams

Reply:
WE cant do that if the path requires the session id.

Reply:
We agreed not to support session_id here